src/main/java/org/spongepowered/api/text/Texts.java

```diff
@@ -175,25 +203,25 @@ public static Text of() {
      * @return The built text object
      */
     public static Text of(Object... objects) {
+        // Shortcut for lonely TextRepresentables
+        if (objects.length == 1 && objects[0] instanceof TextRepresentable) {
```

Comment:
On one hand he may not now that it supports that on the other hand Texts.joinOf uses this method and you may not want to convert that list of X to Text in the first hand. Especially if you pass that list to the Texts#format method as replacement.
.
This tiny piece of code also allows me to display all `CatalogType`s as a list including `TextColor`s. I don't know whether my Test plugin still works, but it could show you this.